common/changes/@itwin/core-quantity/nam-formats-provider_2025-04-11-16-02.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-quantity",
+      "comment": "Add FormatDefinition, FormatsProvider interfaces to core-quantity ",
```

- Code in backticks.
- No need to name the package since this is package-specific change file.

```suggestion
      "comment": "Add `FormatDefinition`, `FormatsProvider` and `MutableFormatsProvider` interfaces.",
```

---

done